final_task/rss_reader/Entry.py

```diff
@@ -0,0 +1,77 @@
+import logging
+
+
+class Entry:
+    """class for every article from http:link...link.rss"""
+
+    def __init__(self, title: str, date: str, article_link: str, summary: str, links: list):
+        self.title = title
+        self.date = date
+        self.article_link = article_link
+        self.summary = summary
+        self.links = links
+        self.parse_html()
+        logging.info("Entry object created")
+
+    def print_title(self) -> None:
+        logging.info("function \"print_title\" started")
+
+        print("Title: ", self.title)
+
```

Comment:
Here and below: you do not really need empty lines here. Code will be still pretty readable without them

Reply:
fixed